metricbeat/docs/how-metricbeat-works.asciidoc

```diff
@@ -18,7 +18,8 @@ image::./images/module-overview.png[Modules and metricsets]
 Likewise, the MySQL module provides a `status` metricset that collects data
 from MySQL by running a http://dev.mysql.com/doc/refman/5.7/en/show-status.html[`SHOW GLOBAL STATUS`]
 SQL query. Metricsets make it easier for you by grouping sets of related metrics together
-in a single request returned by the remote server.
+in a single request returned by the remote server. A certain number of metricsets in each module
```

Comment:
How about something a bit shorter:

Each module has default metricsets that are enabled if there are no user-enabled metricsets.